### ISSUE TITLE
Invalid Credentials (https://azuresynapsestorage.blob.core.windows.net/sampledata)

### ISSUE THREAD

**Issue by jhrobinson** [PacktPublishing/Learn-Microsoft-Fabric#6]
On page 140, I am supposed to connect to https://azuresynapsestorage.blob.core.windows.net/sampledata.  When I follow the steps, I receive an "Invalid credentials" error.  Why is that?

**Comment by mike-westaway** [comment]
There is an article showing how to load sample data, perhaps refer to this instead.
https://learn.microsoft.com/en-us/fabric/data-factory/tutorial-load-sample-data-to-data-warehouse

**Comment by drobert-bfm** [comment]
@mike-westaway This would require a rewrite of much of the book. A better approach would be to provide the dataset in some other, freely accessible repository, or one under the control of the authors, and to provide an errata showing how to load that dataset instead.

**Comment by kelvin519** [comment]
yes, I agree, the author should rectify the sample data accessibility, else we have difficulty following through the examples. I am only on page 72 of the book and cannot load in the data from https://azuresynapsestorage.blob.core.windows.net/sampledata   this is obviously obsolete.

<img width="734" alt="Image" src="https://github.com/user-attachments/assets/35856bc6-3bed-4a2c-b8da-f8ce71e55139" />

I would suggest the reader follows the other link https://learn.microsoft.com/en-us/fabric/data-engineering/tutorial-lakehouse-data-ingestion which would be able to load in the data properly.
